pkg/bootstrapper/bootstrapper.go

```diff
@@ -46,7 +46,7 @@ const (
 	// this is used to parse the kubelet args
 	kubeletSystemdName = "kubelet.service"
 	// kubeletPauseContainerImage is the location of the image we will use for the kubelet pause container
-	kubeletPauseContainerImage = "mcr.microsoft.com/oss/kubernetes/pause:3.4.1"
+	kubeletPauseContainerImage = "mcr.microsoft.com/oss/kubernetes/pause:3.6"
```

Comment:
Best to always stay at the latest to catch base image fixes.

Reply:
I don't think pause images are tied to a specific kube version

Reply:
I thought they might be because of this doc: https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#pause-container 
But makes sense to have latest. I'm ok with 3.6

Reply:
We should keep in mind we are moving to the version that adds support for 2022. 3.5 might not support it.